Dockerfile.ubuntu

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:20.04
+
+#Timezone settings
+ENV TZ=Europe/Uzhgorod
```

Comment:
Uzhgorod should be deleted. Add London instead. You don't need to expose where you live...

Reply:
For sure. It's not my town, but anyway.